_includes/election-summary.html

```diff
@@ -11,10 +22,57 @@ <h2>Top 3 Most Expensive Races</h2>
     <div class="l-section__content">
       {% for race in data_totals['most_expensive_races'] %}
         <div>
-          <a href="/{{race['type']}}/{{ballot.locality}}/{{ballot.slug}}/{{race['slug']}}/">{{race['title']}}</a>
-          <span class="election-summary__race-amount">{{race['amount'] | dollars}}</span>
+          <a href="/{{race.type}}/{{ballot.locality}}/{{ballot.slug}}/{{race.slug}}/">{{race['title']}}</a>
+          <span class="election-summary__race-amount">{{ race.amount | dollars }}</span>
         </div>
       {% endfor %}
     </div>
   </section>
+  <section class="l-section">
+    <div class="l-section__content">
+      <h2>Who is contributing to races on this ballot?</h2>
+    </div>
+    <div class="l-section__content">
+      {% for contribution_type in election_totals.contributions_by_type %}
```

Comment:
Above, you have this line written as: 
```
{% for contribution_type in election_totals['contributions_by_type'] %}
```
Is there a reason for the difference?

Reply:
No reason. I don't know why we wound up with a mix of square bracket and dot notation in this file. Dot would be consistent with the rest of the code, I think

Reply:
Great! I ❤️ consistency.